bin/pt-stalk

```diff
@@ -128,7 +128,6 @@ OPT_ERRS=0        # How many command line option errors
 OPT_VERSION=""    # If --version was specified
 OPT_HELP=""       # If --help was specified
 OPT_ASK_PASS=""   # If --ask-pass was specified
-OPT_MYSQL_ONLY="" # If --mysql-only was specified
```

Comment:
why did you remove this option ? you added another one so you should add it here. Probably it was missed from your previous commits.

Reply:
Function _eval_po from parse_options.sh evaluates such options automatically and assigns value of mysql-only to OPT_MYSQL_ONLY, value system-only to OPT_SYSTEM_ONLY, and so on